menu.py

```diff
@@ -242,15 +243,11 @@ def draw(self):
                             pyxel.text(25, 50, "Objective:", pyxel.COLOR_WHITE)
                             pyxel.text(30, 60, "Take control of all settlements", pyxel.COLOR_WHITE)
                             pyxel.line(24, 70, 175, 70, pyxel.COLOR_GRAY)
-                            pyxel.text(25, 75, "Like any strong leader, you want the", pyxel.COLOR_WHITE)
-                            pyxel.text(25, 81, "best for your people. However,", pyxel.COLOR_WHITE)
-                            pyxel.text(25, 87, "constant attacks by filthy Heathens", pyxel.COLOR_WHITE)
-                            pyxel.text(25, 93, "and enemy troops are enough to wear", pyxel.COLOR_WHITE)
-                            pyxel.text(25, 99, "any great leader down. It is time to", pyxel.COLOR_WHITE)
-                            pyxel.text(25, 105, "put an end to this, and become the", pyxel.COLOR_WHITE)
-                            pyxel.text(25, 111, "one true empire. Other empires will", pyxel.COLOR_WHITE)
-                            pyxel.text(25, 117, "wither at your blade, and they will", pyxel.COLOR_WHITE)
-                            pyxel.text(25, 123, "be all the more thankful for it.", pyxel.COLOR_WHITE)
+                            self.draw_paragraph(25, 75, """Like any strong leader, you want the best for your people. However,
```

Comment:
Would advise running pylint, this line and many others go over the 120 char limit :(

Reply:
[841bd66](https://github.com/ChrisNeedham24/microcosm/pull/91/commits/841bd66dbd0cc1c655d54ad409b59603d697ddd2)